README.md

```diff
@@ -2,7 +2,7 @@
 
 These DockerFiles enable running PowerShell in a container for each Linux distribution we support.
 
-This requires an up-to-date version of Docker, such as 1.12.
+This requires Docker 17.05 or newer.
```

Comment:
Where is the 17.05 number coming from?

Reply:
```
 docker version
Client:
 Version:      18.03.1-ce
 API version:  1.37
 Go version:   go1.9.5
 Git commit:   9ee9f40
 Built:        Thu Apr 26 07:13:02 2018
 OS/Arch:      darwin/amd64
 Experimental: true
 Orchestrator: swarm

Server:
 Engine:
  Version:      18.03.1-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.5
  Git commit:   9ee9f40
  Built:        Thu Apr 26 07:22:38 2018
  OS/Arch:      linux/amd64
  Experimental: true
```

Reply:
https://docs.docker.com/release-notes/docker-ce/#17050-ce-2017-05-04 says

17.05.0-ce (2017-05-04):
* Allow using build-time args (ARG) in FROM https://github.com/docker/docker/pull/31352